.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: go
 sudo: false
+before_script:
```

Comment:
How about moving this down so it's placed above `script`?

```yaml
before_script:
- go vet ./...
script:
- ./script/coverage
- ./script/format
```

Reply:
Yes I will work on cleaning those errors. Thanks